app/uk/gov/hmrc/childcarecalculatorfrontend/models/schemes/EmploymentSupportedChildcare.scala

```diff
@@ -25,10 +25,13 @@ class EmploymentSupportedChildcare extends Scheme {
 
   override def eligibility(answers: UserAnswers): Eligibility = {
     val No = YesNoUnsureEnum.NO.toString
+    val NotSure = YesNoUnsureEnum.NOTSURE.toString
+
 
     val hasParentChildcareCosts: Boolean = answers.childcareCosts.contains(YesNoNotYetEnum.YES.toString)
-    val hasPartnerChildcareVouchers = answers.partnerChildcareVouchers.fold(false)(x => !x.equals(No))
-    val hasParentChildcareVouchers = answers.yourChildcareVouchers.fold(false)(x => !x.equals(No))
+    val hasPartnerChildcareVouchers = answers.partnerChildcareVouchers.fold(false)(x => !(x.equals(No)|| x.equals(NotSure)))
```

Comment:
!(x.equals(No)) is inverted logic, more readable to have x.equals(Yes)?

Reply:
Done